spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -1862,6 +1862,8 @@ The following topics provide more details:
 * <<api-guide-resources-task-executions-stopping>>
 * <<api-guide-resources-task-executions-list>>
 * <<api-guide-resources-task-executions-list-by-name>>
+* <<api-guide-resources-task-thin-executions-list>>
```

Comment:
On the source port, there was some concerns about the docs not rendering properly ([here](https://github.com/spring-cloud/spring-cloud-dataflow/pull/5994/files#r1813314218)). Can you verify this has been fixed or is not an issue here as well?

Reply:
I checked the rendering on the docs.